### PR TITLE
Add shopware 5.6.3 dockware dev version

### PR DESCRIPTION
**Pull request by frederikdengler**
Just adding the dev docker image for shopware 5.6.3 version.

### CODE REVIEW

File: variants/dev/5.6.3/variables.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    "shopware": {
+        "version": "5.6.3",
+        "download_url": "https://www.shopware.com/de/Download/redirect/version/sw5/file/install_5.6.3_fec7645a72a0393f8a39f48ddd6c27e138f44513.zip"
+    },
+    "composer": {
+        "version": 1
+    },
+    "php": {
+        "default_version": "7.0",
```
Comment (martinweinmayr) [inline variants/dev/5.6.3/variables.json:14]: pls set this version to 7.2 as 7.0 is not supported in this version, this leads to the build error

File: manifest.json [hunk 2]
```diff
@@ -110,6 +110,7 @@
             "dev/5.6.7:5.6.7",
             "dev/5.6.6:5.6.6",
             "dev/5.6.4:5.6.4",
+            "dev/5.6.3:5.6.3",
```
Comment (martinweinmayr) [inline manifest.json:113]: could you pls also add the play version, as we always have both dev and play version.

**Review by martinweinmayr — COMMENTED** [review]
pls resolve the 2 comments than i can merge it, many thanks

File: variants/dev/5.6.3/variables.json (hunk 1 above)
Comment (frederikdengler) [reply variants/dev/5.6.3/variables.json]: Is now set to 7.2

File: manifest.json (hunk 2 above)
Comment (frederikdengler) [reply manifest.json:114]: Also play docker is for 5.6.3 now available

**Review by martinweinmayr — APPROVED** [review]

**Pull request merged**